public/app/features/dashboard-scene/conditional-rendering/ConditionalRenderingEditor.tsx

```diff
@@ -11,13 +12,24 @@ export function useConditionalRenderingEditor(
     return null;
   }
 
+  const title = t('dashboard.conditional-rendering.root.title', 'Show/hide rules');
+
   return new OptionsPaneCategoryDescriptor({
-    title: t('dashboard.conditional-rendering.title', 'Conditional rendering options'),
+    title,
     id: 'conditional-rendering-options',
+    renderTitle: () => (
+      <Stack direction="row" gap={1} alignItems="center">
+        <div>{title}</div>
+        <Icon name={!conditionalRendering.evaluate() ? 'eye-slash' : 'eye'} />
+        <Tooltip content={conditionalRendering.info}>
+          <Icon name="info-circle" />
+        </Tooltip>
+      </Stack>
```

Comment:
Can we skip this info icon? And just put the tooltip around the eye icon? the double icons looks a bit messy 

![image](https://github.com/user-attachments/assets/3c930719-b571-448a-81ca-afceec3f8d31)

Reply:
Also need some white space around `/` => `Show / hide`

Reply:
Done ^^